packages/react/rollup.config.js

```diff
@@ -56,10 +74,66 @@ const getConfig = (format) => ({
       },
     }),
     terser(),
+    extractCSS ? cssText() : undefined,
+    extractCSS ? insertCSS() : undefined,
+    // extractCSS
+    //   ? del({
+    //       targets: 'lib/tmp',
+    //       hook: 'closeBundle',
+    //       verbose: true,
+    //     })
+    //   : undefined,
   ],
   external: getExternal(format),
 });
 
+const esmInput = {
+  index: 'src/index.ts',
+  'components/index': 'src/components/index.ts',
+  'icons/index': 'src/icons/index.ts',
+  'ssr/index': 'src/ssr/index.ts',
+  'components/Accordion/index': 'src/components/accordion/index.ts',
+  'components/Button/index': 'src/components/button/index.ts',
+  'components/Card/index': 'src/components/card/index.ts',
+  'components/Checkbox/index': 'src/components/checkbox/index.ts',
+  'components/Columns/index': 'src/components/columns/index.ts',
+  'components/Combobox/index': 'src/components/dropdown/combobox/index.ts',
+  'components/Container/index': 'src/components/container/index.ts',
+  'components/DateInput/index': 'src/components/dateInput/index.ts',
+  'components/Dialog/index': 'src/components/dialog/index.ts',
+  'components/ErrorSummary/index': 'src/components/errorSummary/index.ts',
+  'components/Fieldset/index': 'src/components/fieldset/index.ts',
+  'components/FileInput/index': 'src/components/fileInput/index.ts',
+  'components/Footer/index': 'src/components/footer/index.ts',
+  'components/ImageWithCard/index': 'src/components/imageWithCard/index.ts',
+  'components/Koros/index': 'src/components/koros/index.ts',
+  'components/Link/index': 'src/components/link/index.ts',
+  'components/Linkbbox/index': 'src/components/linkbox/index.ts',
+  'components/LoadingSpinner/index': 'src/components/loadingSpinner/index.ts',
+  'components/Logo/index': 'src/components/logo/index.ts',
+  'components/Navigation/index': 'src/components/navigation/index.ts',
+  'components/Notification/index': 'src/components/notification/index.ts',
+  'components/NumberInput/index': 'src/components/numberInput/index.ts',
+  'components/PasswordInput/index': 'src/components/passwordInput/index.ts',
```

Comment:
The react pagination is not yet released, so that is why it is missing. I think we could modify the `yarn scaffold` to add the react component to this file also.

Reply:
Maybe 👍